datus/tools/db_tools/duckdb_connector.py

```
@@ -40,23 +43,354 @@ def _metadata_names(_type: str) -> _DBMetadataNames:
+        """Execute an INSERT SQL statement."""
+        try:
+            self.connect()
+            result = self.connection.execute(sql)
+            row_count = result.fetchone()[0] if result else 0
+            return ExecuteSQLResult(
+                success=True,
+                sql_query=sql,
+                sql_return=str(row_count),
+                row_count=row_count,
+            )
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)